fbw-common/src/wasm/systems/systems/src/shared/mod.rs

```diff
@@ -627,6 +627,20 @@ impl DelayedFalseLogicGate {
         }
     }
 
+    pub fn starting_as(mut self, state: bool) -> Self {
+        self.set_output(state);
+        self
+    }
+
+    fn set_output(&mut self, state: bool) {
+        self.expression_result = state;
+        if !state {
+            self.false_duration = self.delay;
+        } else {
+            self.false_duration = Duration::from_millis(0)
+        }
```

Comment:
```suggestion
        self.false_duration = if !state {
            self.delay
        } else {
            Duration::from_millis(0)
        };
```
According to coding [guidelines](https://github.com/flybywiresim/aircraft/blob/master/fbw-common/src/wasm/systems/guidelines.md#single-assignment-for-simple-if-else).